vim25/soap/client.go

```diff
@@ -353,35 +353,39 @@ func (c *Client) SetCertificate(cert tls.Certificate) {
 	// Extension certificate
 	t.TLSClientConfig.Certificates = []tls.Certificate{cert}
 
-	// Proxy to vCenter host on port 80
-	host, _ := splitHostPort(c.u.Host)
+	/*
+		WHY?
```

Comment:
Not exactly sure why proxy it for client requests with certificate. needs to discuss it with Doug.